scripts/zones/Southern_San_dOria/npcs/Kipopo.lua

```diff
@@ -6,18 +6,41 @@
 -----------------------------------
 require("scripts/globals/status");
 require("scripts/globals/crafting");
+require("scripts/globals/quests");
 local ID = require("scripts/zones/Southern_San_dOria/IDs");
 -----------------------------------
 
 function onTrade(player,npc,trade)
+    local sayItWithAHandbagCS = player:getVar("sayItWithAHandbagCS")
+
+    if (player:hasKeyItem(dsp.ki.TORN_PATCHES_OF_LEATHER) and sayItWithAHandbagCS == 2 and trade:getItemCount() == 3) then
+        if (trade:hasItemQty(2012,1) and trade:hasItemQty(850,1) and trade:hasItemQty(816,1)) then
+        player:startEvent(910)
+        end
+    end
 end;
 
 function onTrigger(player,npc)
+    local sayItWithAHandbag = player:getQuestStatus(CRYSTAL_WAR, dsp.quest.id.crystalWar.SAY_IT_WITH_A_HANDBAG)
+    local sayItWithAHandbagCS = player:getVar("sayItWithAHandbagCS")
     local guildMember = isGuildMember(player,7);
     local SkillCap = getCraftSkillCap(player, dsp.skill.LEATHERCRAFT);
     local SkillLevel = player:getSkillLevel(dsp.skill.LEATHERCRAFT);
 
-    if (guildMember == 1) then
+    if (sayItWithAHandbag == QUEST_COMPLETED and sayItWithAHandbagCS == 4 and player:hasItem(19110)) then
```

Comment:
`if sayItWithAHandbag == QUEST_COMPLETED then` should be sufficient by itself~

Reply:
@ibm2431  this check is just like mentioned on ```What's this 5 being used for?``` Checking if CS == 4 to play this one, also to play this cutscene the item needs to be on the user's inventory, otherwise it won't play. I'm not sure how accurate that is to retail but it seems plausible as Kipopo refers to the item in the cutscene 😿

Reply:
changed to ```sayItWithAHandbag == QUEST_COMPLETED and sayItWithAHandbagBonus CS == 1 and player:hasItem(19110)``` (See comment above for comment regarding player:hasItem)